pkg/apis/resources/v1alpha1/register.go

```diff
@@ -36,8 +36,10 @@ func Resource(resource string) schema.GroupResource {
 }
 
 var (
+	// SchemeBuilder is a new Scheme Builder which registers our API.
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
+	// AddToScheme is a reference to the Scheme Builder's AddToScheme function.
```

Comment:
An `install` package is usually only used when multiple API versions exist in a certain group (e.g. `core` has an internal version, `v1alpha1` and `v1beta1`). This helps to quickly add all of them to a scheme. It doesn't make much sense to have it for an API group which currently (and for the foreseeable future) only has one version. So, I would not add it now.
I see that the `extensions` group has such `install` package as well, however, I would rather delete it since it doesn't provide much value compared to the normal `AddToScheme` function.

Reply:
> I see that the extensions group has such install package as well, however, I would rather delete it since it doesn't provide much value compared to the normal AddToScheme function.

Do you plan to do this as part of this PR or in a separate one?

Reply:
Done now with 153eea7